src/test/java/picard/sam/markduplicates/DuplicationMetricsTest.java

```diff
@@ -22,11 +22,12 @@
  * THE SOFTWARE.
  */
 
-package picard.sam;
+package picard.sam.markduplicates;
 
 import org.testng.Assert;
 import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
+import picard.sam.DuplicationMetrics;
```

Comment:
Is there a reason to move the test but not the class? Normally the test for a class is in the same package as the class being tested.

Reply:
oops

Reply:
Does that mean you will move it back? It looks like this wasn't addressed.

Reply:
yeah. not sure what happened there...sorry.